python/lsst/obs/cfht/makeMegacamRawVisitInfo.py

```diff
@@ -40,8 +40,12 @@ class MakeMegacamRawVisitInfo(MakeRawVisitInfo):
     def setArgDict(self, md, argDict):
         """Set an argument dict for VisitInfo and pop associated metadata
 
-        @param[in,out] md  metadata, as an lsst.daf.base.PropertyList or PropertySet
-        @param[in,out] argdict  a dict of arguments
```

Comment:
These parameters are marked `in,out` but the new version of the docstring doesn't indicate that the arguments can be modified. Not sure what the standard numpydoc approach is.

Reply:
Just `Parameters`. If it's important to talk about how a parameter will be modified you can do that in a description, otherwise, in Python, it's generally assumed the thing you're passing an argument to is not making a copy, but rather sharing the object.

Reply:
I think it's polite to say if something is going to be added to a dict.